installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh

```diff
@@ -40,7 +55,50 @@ do
     service_name=$element
   fi
 
-  if [ $element == "jazz_email" ] || [ $element == "jazz_usermanagement" ] || [ $element == "jazz_codeq" ] || [ $element == "jazz_metrics" ]; then
+  if [[ $(inArray "${lambda_services[@]}" "$element") ]]; then
+	 if [[ $(inArray "${runtime_six_services[@]}" "$element") ]]; then
```

Comment:
increase indentation of this `if .. else ` condition

Reply:
done